float64[] state
---
float32[] action
bool success